# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/ash/src/vk.rs
# Commit: ef5dc3e04e5f
# Full Hash: ef5dc3e04e5f271eea0636ab3a495e95cc912f1d
# Author: Dzmitry Malyshau <dmalyshau@mozilla.com>
# Date: 2021-09-04 09:40:24
# Regressor Bug: 1726626
# File Overlap Count: 3
# Description:
#   Bug 1726626 - Move gfx/wgpu into a 3rd party dependency r=jgilbert,bholley
#   
#   This update makes wgpu a vendored dependency instead of having it in gfx/wgpu.
#   
#   ## Notes
# ==============================================================================

diff --git a/third_party/rust/ash/src/vk.rs b/third_party/rust/ash/src/vk.rs
--- a/third_party/rust/ash/src/vk.rs
+++ b/third_party/rust/ash/src/vk.rs
@@ -19,16 +19,20 @@ pub use definitions::*;
 mod enums;
 pub use enums::*;
 mod extensions;
 pub use extensions::*;
 mod feature_extensions;
 pub use feature_extensions::*;
 mod features;
 pub use features::*;
+#[doc = r" Native bindings from Vulkan headers, generated by bindgen"]
+#[allow(nonstandard_style)]
+#[allow(deref_nullptr)]
+pub mod native;
 mod platform_types;
 pub use platform_types::*;
 #[doc = r" Iterates through the pointer chain. Includes the item that is passed into the function."]
 #[doc = r" Stops at the last `BaseOutStructure` that has a null `p_next` field."]
 pub(crate) unsafe fn ptr_chain_iter<T>(ptr: &mut T) -> impl Iterator<Item = *mut BaseOutStructure> {
     let ptr: *mut BaseOutStructure = ptr as *mut T as _;
     (0..).scan(ptr, |p_ptr, _| {
         if p_ptr.is_null() {